src/main/java/core/basesyntax/CanDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface CanDraw {
```

Comment:
```suggestion
interface Drawable {
```

Reply:
https://mate-academy.github.io/style-guides/java/java.html#:~:text=Interface%20names%20may%20also%20be%20nouns%20or%20noun%20phrases%20(for%20example%2C%20List)%2C%20but%20may%20sometimes%20be%20adjectives%20or%20adjective%20phrases%20instead%20(for%20example%2C%20Readable).